components/eventsrv/src/subscriber.rs

```diff
@@ -41,10 +49,11 @@ fn main() {
                 let received_payload = String::from_utf8(event.get_payload().to_vec()).unwrap();
                 let member_id = event.get_member_id();
                 let timestamp = event.get_timestamp();
-
-                println!("Timestamp {}", timestamp);
-                println!("Member ID {}\n", member_id);
-                debug!("{}\n", received_payload);
+                let service = event.get_service();
+                println!("SUBSCRIBER: Timestamp {}", timestamp);
+                println!("SUBSCRIBER: Member ID {}", member_id);
+                println!("SUBSCRIBER: Service {}", service);
+                println!("SUBSCRIBER: Payload {}\n", received_payload);
```

Comment:
@christophermaier From what I've seen so far, it's common to use `debug!` for noisy output that doesn't need to be seen by the end user, but would be helpful when debugging.  If it doesn't need to be seen by the end user _and_ it's also not helpful when debugging, then just get rid of it.